src/include/common/types/value.h

```diff
@@ -109,6 +109,13 @@ class NodeVal {
         properties.emplace_back(key, std::move(value));
     }
 
+    inline const vector<pair<std::string, unique_ptr<Value>>>& getProperties() const {
+        return properties;
+    }
+
+    inline const Value& getNodeID() const { return *idVal; }
```

Comment:
We can probably directly do `inline const nodeID_t getNodeID() const { return idVal.getValue<nodeID_t>()}` since we know the data type. Ditto for getLabelName().